prototype/distribution/features/northbound-ogc-sensorthings-mqtt-feature/src/main/resources/northbound-ogc-sensorthings-mqtt-feature.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "org.eclipse.sensinact.gateway.distribution.features:northbound-ogc-sensorthings-mqtt-feature:osgifeature:0.0.2-SNAPSHOT",
+  "bundles": [
+    { "id": "io.moquette:moquette-broker:0.16" },
```

Comment:
I don't think that this is a bundle. I repackage it inside the northbound implementation.
```suggestion
```